tools/realsense-viewer/openvino-face-detection.cpp

```diff
@@ -225,6 +226,7 @@ class openvino_face_detection : public post_processing_worker_filter
         }
         // A depth frame is optional: if not enabled, we won't get it, and we simply won't provide depth info...
         auto df = fs.get_depth_frame();
```

Comment:
Same here. Did you check it with live cam?

Reply:
yes sure, no exception is thrown when fs is NULL, it will simply return NULL to df and that's what we need